### ISSUE TITLE
Building errors

### ISSUE THREAD

**Issue by Col-blimp** [libragnar/ragnar#93]
Hi, I keep getting the same two build errors when compiling.

Using VS 2013 free edition on Win 8.1 x64.

Log:  http://pastebin.com/nzfx8B2W

Thanks.

**Comment by vktr** [comment]
Did you run the `bootstrap.bat` file? If so, did you get any errors?

**Comment by Col-blimp** [comment]
Hi there, yes this:

>.\run-bootstrap.ps1 : The term '.\run-bootstrap.ps1' is not recognized as the
>name of a cmdlet, function, script file, or operable program. Check the
>spelling of the name, or if a path was included, verify that the path is
>correct and try again.
>At line:1 char:1
>+ .\run-bootstrap.ps1
>+ ~~~~~~~~~~~~~~~~~~~
>   + CategoryInfo          : ObjectNotFound: (.\run-bootstrap.ps1:String) [],
>    CommandNotFoundException
 >   + FullyQualifiedErrorId : CommandNotFoundException
>
>C:\Windows\system32>pause
>Press any key to continue . . .

but I just ran the Powershell script directly and it worked with no errors.

**Comment by Col-blimp** [comment]
I re-downloaded and started again and ir works now, thanks.

This is probably a stupid question but to add this to my own project out the Ragnar.Client\bin\Debug folder what files do I need? Using VS C++ 2013.

**Comment by vktr** [comment]
You will need all `boost*` files as well as `torrent.dll` and `Ragnar.dll`.

**Comment by Col-blimp** [comment]
Thanks, looking at the functions of Ragnar.dll with dllexp.exe then it shows none so how do I initialise the wrapper in VS?

**Comment by vktr** [comment]
`Ragnar.dll` is a managed assembly.

The readme contains a small usage example, and the test client shows a bit more.

**Comment by Col-blimp** [comment]
Thanks, the nuget package, does this need to be installed on each machine the client will use? Sorry about these questions as I was using a Lua wrapper before and it was just a case of calling 'require torrent.dll' to put all the wrapper functions on the stack.

**Comment by vktr** [comment]
Yes. The nuget package contains everything you need to use Ragnar and the packaged binaries will need to be deployed to wherever the client will be used. If you are not interested in developing for Ragnar, the nuget package is the recommended way of using it.

Create a new project, install the nuget package, compile. All binaries in the `bin/[Debug|Release]` output folder needs to be deployed to use the client.

**Comment by Col-blimp** [comment]
Hi again :)

So I get this error when installing the nuget:

>Install failed. Rolling back...
>Install-Package : Could not install package 'Ragnar 0.4.0'. You are trying to install this package into >a project that 
>targets 'Native,Version=v0.0', but the package does not contain any assembly references or >content files that are 
>compatible with that framework. For more information, contact the package author.
>At line:1 char:1
>+ Install-Package Ragnar -Version 0.4.0
>+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>   + CategoryInfo          : NotSpecified: (:) [Install-Package], InvalidOperationException
>   + FullyQualifiedErrorId : >NuGetCmdletUnhandledException,NuGet.PowerShell.Commands.InstallPackageCommand

**Comment by vktr** [comment]
You need to install the nuget into a managed (.NET v4.5) project.

**Comment by Col-blimp** [comment]
Ahh, so its not compatible with a C++ Vs project then.

**Comment by vktr** [comment]
No. If you're writing a C++ project I can recommend using Rasterbar-libtorrent directly.
